src/actions/menu.actions.ts

```
@@ -81,32 +64,37 @@ const getModuleMenuItems = async (): Promise<MenuItemMeta[]> => {
+
+    const composeMenuGroup = getMenuGroupComposer(t);
+    const composeModuleMenuItem = getModuleMenuItemComposer(t);
+
+    return Object.entries(groups)
+      .reduce((acc: MenuGroup[], [group, modules]) => {
+        if (!modules) {
+          return acc;
+        }
+
+        if (modules.length === 1) {
+          return [...acc, composeModuleMenuItem(modules[0], profileArea)];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)